packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -9,66 +9,187 @@
   justify-items: stretch;
 }
 
-/* == Image styles == */
-.result-root.image-large,
-.result-root.image-small {
-  grid-template-areas:
-    'visual badges          .               actions'
-    'visual title           title           title'
-    'visual title-metadata  title-metadata  title-metadata'
-    'visual emphasized      emphasized      emphasized'
-    'visual excerpt         excerpt         excerpt'
-    'visual bottom-metadata bottom-metadata bottom-metadata';
-  grid-template-columns: minmax(0, min-content) auto 1fr auto;
-  grid-template-rows: auto auto auto auto auto auto;
-}
+/* == Divider == */
+.result-root {
+  border-top: 1px solid var(--atomic-neutral);
 
-.result-root.image-large atomic-result-section-visual {
-  width: 20.375rem;
-  height: 20.375rem;
-}
+  &.first {
+    border-top: none;
+  }
 
-.result-root.image-small atomic-result-section-visual {
-  width: 10rem;
-  height: 10rem;
+  &.image-small,
+  &.image-icon,
+  &.image-none {
+    @media only screen and (max-width: 640px) {
+      border-top: none;
+    }
+  }
 }
 
-.result-root.image-icon {
-  grid-template-areas:
-    'badges badges          .               actions'
-    'visual title           title           title'
-    'visual title-metadata  title-metadata  title-metadata'
-    'visual emphasized      emphasized      emphasized'
-    'visual excerpt         excerpt         excerpt'
-    'visual bottom-metadata bottom-metadata bottom-metadata';
-  grid-template-columns: minmax(0, min-content) auto 1fr auto;
-  grid-template-rows: auto auto auto auto auto auto;
-}
+/* == Image styles == */
+:host .result-root.density-comfortable,
+:host .result-root.density-normal,
+:host .result-root.density-compact {
+  &.image-large {
+    grid-template-areas:
+      'visual badges          .               actions'
+      'visual title           title           title'
+      'visual title-metadata  title-metadata  title-metadata'
+      'visual emphasized      emphasized      emphasized'
+      'visual excerpt         excerpt         excerpt'
+      'visual bottom-metadata bottom-metadata bottom-metadata'
+      'visual .               .               .';
+    grid-template-columns: minmax(0, min-content) auto 1fr auto;
+    grid-template-rows: repeat(6, auto) 1fr;
+
+    @media only screen and (max-width: 640px) {
```

Comment:
heads up for your media queries https://tailwindcss.com/docs/breakpoints
Tailwind is mobile first meaning the css is applied as the page grow `@media (min-width: 640px) { ... }`

We should either:
- emulate what their doing
- or lower the `max-width: 640px` to `max-width: 639px` 

so there's no buggy width where some mobile styles are applied and some aren't